baml_language/crates/baml_hir/src/item_tree.rs

```diff
@@ -0,0 +1,263 @@
+//! Position-independent item storage.
+//!
+//! The `ItemTree` contains minimal representations of all items in a container.
+//! It acts as an "invalidation barrier" - only changes to item signatures
+//! cause the `ItemTree` to change, not edits to whitespace, comments, or bodies.
```

Comment:
Curious - why don't changes to bodies (does this mean function bodies) cause the function to get invalidated?